src/domain/home/Homepage.jsx

```diff
@@ -1,27 +1,47 @@
 import React, { useState, useEffect } from 'react';
-import { Intro, TheApp, ActiveCampaigns, HowItWorks } from './components/index';
-import classes from './Homepage.module.scss';
+import {
+    Intro,
+    TakeAction,
+    ActiveCampaigns,
+    WaveBreak,
+    HowItWorks,
+    Partners,
+    Newsletter,
+} from './components/index';
 
 const Homepage = () => {
-  const [campaigns, setCampaigns] = useState([]);
+    const [campaigns, setCampaigns] = useState([]);
+    const [partners, setPartners] = useState([]);
 
-  useEffect(() => {
-    const fetchCampaigns = async () => {
-      const resp = await fetch('https://api.now-u.com/api/v1/campaigns');
-      const campaigns = await resp.json();
-      setCampaigns(campaigns?.data);
-    };
-    fetchCampaigns();
-  }, []);
+    useEffect(() => {
+        const fetchCampaigns = async () => {
+            const resp = await fetch('https://api.now-u.com/api/v1/campaigns');
```

Comment:
Using the `campaignsURL` defined in [`constants.js`](https://github.com/now-u/now-u-campaigns/blob/1f17ad8c908c7c4b2c3c3833add174c002dc9c9c/src/utils/constants.js#L25) would be better. If the API URL changes, we can just change in the constants file: